sockeye/rerank.py

```diff
@@ -102,18 +136,36 @@ def rerank(args: argparse.Namespace):
             utils.check_condition('translations' in hypotheses,
                                   "Reranking requires nbest JSON input with 'translations' key present.")
             num_hypotheses = len(hypotheses['translations'])
+            source = hypotheses['text']
 
             if not num_hypotheses > 1:
                 logger.info("Line %d contains %d hypotheses. Nothing to rerank.", i, num_hypotheses)
                 reranked_hypotheses = hypotheses
             else:
-                reranked_hypotheses = reranker.rerank(hypotheses, reference)
+                reranked_hypotheses = reranker.rerank(hypotheses,
+                                                          reference,
+                                                          source,
```

Comment:
if `source` is a field in `hypotheses`, why change the signature of this function?